doc/man3/SSL_CTX_set_tlsext_servername_callback.pod

```diff
@@ -39,16 +41,25 @@ SSL_get_servername_type() returns the servername type or -1 if no servername
 is present. Currently the only supported type (defined in RFC3546) is
 B<TLSEXT_NAMETYPE_host_name>.
 
+SSL_set_tlsext_host_name() sets the server name indication extension typically
+in ClientHello with value B<name>. The type of server name indication extension
```

Comment:
Oh, no, this is nongrammatical.  How about

SSL_set_tlsext_host_name() sets the server name indication ClientHello extension to contain the value B<name>. 
?